### PR TITLE
add keras docs by manfei

### PR DESCRIPTION
**Pull request by LudviqIntel**

### CODE REVIEW

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+## **RReLU**
+Applies the randomized leaky rectified linear unit element-wise to the input.
+f(x) = max(0,x) + a * min(0, x) where a ~ U(l, u).
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: add a new line. See the rendered results of GitHub and website.

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+**Scala example:**
+```scala
+import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
+import com.intel.analytics.zoo.pipeline.api.keras.layers.RReLU
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.tensor.Tensor
+
+val model = Sequential[Float]()
+model.add(RReLU[Float](inputShape = Shape(1, 4)))
+val forwardinput = Tensor[Float](1, 4).rand()
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: change to input?

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+val forwardoutput = model.forward(forwardinput)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: change to output and gradInput?

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 1x4]
+```
+
+**Python example:**
+```python
+import numpy as np
+from zoo.pipeline.api.keras.layers import RReLU
+from zoo.pipeline.api.keras.models import Sequential
+
+rrmodel = Sequential()
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: change to model?

**Comment by hkvision** [comment]
Rebase with master.

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 1 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 2 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 3 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 4 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+rrmodel.add(RReLU(input_shape = (1,4)))
+rrmodel.forward(np.array([1.0, 2.0, -1.0, -2.0]))
```
[the remark this answers is not in the document]
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+If l == u, a RReLU essentially becomes a LeakyReLU.
+
+Regardless of operating in in-place mode a RReLU will internally allocate an input-sized noise tensor to store random factors for negative inputs.
+
+For reference, see [Empirical Evaluation of Rectified Activations in Convolutional Network](http://arxiv.org/abs/1505.00853).
+
+**Scala:**
+```scala
+RReLU(lower=1.0/8, upper=1.0/3, inPlace)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: missing inputShape?

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+```
+**Python:**
+```python
+RReLU(lower, upper, inPlace)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: missing input_shape?

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+**Parameters:**
+
+* `lower`: Lower boundary of the uniform random distribution. Default is 1.0/8.
+* `upper`: Upper boundary of the uniform random distribution. Default is 1.0/3.
+* `inputShape`: A Single Shape, does not include the batch dimension.
+* `T`: The numeric type of parameter(e.g. weight, bias). Only support float/double now.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: no need T

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+**Python:**
+```python
+RReLU(lower, upper, inPlace)
+```
+
+**Parameters:**
+
+* `lower`: Lower boundary of the uniform random distribution. Default is 1.0/8.
+* `upper`: Upper boundary of the uniform random distribution. Default is 1.0/3.
+* `inputShape`: A Single Shape, does not include the batch dimension.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: Python will be a tuple. Refer to Ding Di's layers to modify this

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 6 above)
Comment (hkvision) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: seems there's no inPlace argument. Refer to RReLU.scala in Zoo.

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
+import com.intel.analytics.zoo.pipeline.api.keras.layers.RReLU
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.tensor.Tensor
+
+val model = Sequential[Float]()
+model.add(RReLU[Float](inputShape = Shape(1, 4)))
+val input = Tensor[Float](1, 4).rand()
+val output = model.forward(input)
+val gradInput = model.backward(input,input)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: no need gradInput, just output

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+**Python example:**
+```python
+import numpy as np
+from zoo.pipeline.api.keras.layers import RReLU
+from zoo.pipeline.api.keras.models import Sequential
+
+model = Sequential()
+model.add(RReLU(input_shape = (1,4)))
+input = np.random.random([1, 4])
+model.forward(input)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: output = ...

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+model.backward(input, input)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: remove this

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+```
+Input is:
+```python
+array([[0.36889318, 0.06288461, 0.90168108, 0.0763195 ]])
+```
+Ouput is:
+```python
+array([[0.36889318, 0.06288461, 0.90168107, 0.0763195 ]], dtype=float32)
+
+array([[0.36889318, 0.06288461, 0.90168107, 0.0763195 ]], dtype=float32)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: remove

File: pyzoo/zoo/pipeline/api/keras/layers/normalization.py [hunk 14]
```diff
@@ -1,4 +1,5 @@
 #
+#
```
Comment (hkvision) [inline pyzoo/zoo/pipeline/api/keras/layers/normalization.py]: ??

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 7 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: fix

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 9 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: fix

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 6 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: ok,fix

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 11 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: ok, fix

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+
+For reference, see [Empirical Evaluation of Rectified Activations in Convolutional Network](http://arxiv.org/abs/1505.00853).
+
+**Scala:**
+```scala
+RReLU(lower=1.0/8, upper=1.0/3, inputShape = null)
+```
+**Python:**
+```python
+RReLU(lower=1.0/8, upper=1.0/3, input_shape = None, name=None)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/activation.md]: remove space around `=`, `input_shape=None`

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 16]
```diff
@@ -240,7 +240,7 @@ input: com.intel.analytics.bigdl.tensor.Tensor[Float] =
 Output is:
 ```scala
 output: com.intel.analytics.bigdl.nn.abstractnn.Activity =
-(1,1,.,.) =
+(1,1,.,.)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: ??

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 17]
```diff
@@ -286,3 +286,600 @@ Output is
...
+**Python:**
+```python
+Cropping3D(cropping=((1, 1), (1, 1), (1, 1)), dim_ordering="th", input_shape=None, name=None)
+```
+
+**Parameters:**
+
+* `cropping`: Int tuple of tuple of length 3. How many units should be trimmed off at the beginning and end of the 3 cropping dimensions (i.e. kernel_dim1, kernel_dim2 and kernel_dim3). Default is ((1, 1), (1, 1), (1, 1)).
+* `dimOrdering`: Format of input data. Either 'CHANNEL_FIRST' (dimOrdering='th') or 'CHANNEL_LAST' (dimOrdering='tf'). Default is 'CHANNEL_FIRST'.
+* `inputShape`: A Single Shape, does not include the batch dimension.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: modify this

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 18]
```diff
@@ -286,3 +286,600 @@ Output is
...
+
+**Scala:**
+```scala
+ShareConvolution2D(nbFilter, nbRow, nbCol, init = "glorot_uniform", activation = null, subsample = (1, 1), padH = 0, padW = 0, 
+                   propagateBack = true, dimOrdering = "th", wRegularizer = null, bRegularizer = null, bias = true, inputShape = null)
+```
+**Python:**
+```python
+ShareConvolution2D(nb_filter, nb_row, nb_col, init = "glorot_uniform", activation = None, subsample = (1, 1), pad_h = 0, pad_w = 0,
+                   propagate_back = True, dim_ordering = "th", W_regularizer = None, b_regularizer = None, bias = True, input_shape = None, name = None)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: remove space surrounded by `=`

File: pyzoo/zoo/pipeline/api/keras/layers/normalization.py (hunk 14 above)
Comment (hkvision) [reply pyzoo/zoo/pipeline/api/keras/layers/normalization.py]: revert the modification of this file :)

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 19]
```diff
@@ -286,3 +286,600 @@ Output is
...
+                or 'sigmoid', etc. for simple activations in the factory method.
+* `subsample`: Int array of length 2 corresponding to the step of the convolution in the
+               height and width dimension. Also called strides elsewhere. Default is (1, 1).
+* `padH`: The additional zeros added to the height dimension. Default is 0.
+* `padW`: The additional zeros added to the width dimension. Default is 0.
+* `propagateBack`: Whether to propagate gradient back. Default is true.
+* `dimOrdering`: Format of input data. Please use DataFormat.NCHW (dimOrdering='th').
+* `wRegularizer`: An instance of [[Regularizer]], (eg. L1 or L2 regularization),
+                  applied to the input weights matrices. Default is null.
+* `bRegularizer`: An instance of [[Regularizer]], applied to the bias. Default is null.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: where's the link for Regularizer? Link them to BigDL's website. Also for regularizers in other layers. Can refer to Ding Di's layers.

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 20]
```diff
@@ -283,3 +283,312 @@ Output is
...
+
+Select(1, -1) will give output [3 6]
+
+**Scala:**
+```scala
+Select(dim, index, inputShape = null)
+```
+**Python:**
+```python
+Select(dim, index, input_shape = None, name = None)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: remove space... check for other layers

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 21]
```diff
@@ -283,3 +283,312 @@ Output is
...
+* `inputShape`: Only need to specify this argument when you use this layer as the first layer of a model. For Scala API, it should be a [`Shape`](../keras-api-scala/#shape) object. For Python API, it should be a shape tuple. Batch dimension should be excluded.
+
+**Scala example:**
+```scala
+import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
+import com.intel.analytics.zoo.pipeline.api.keras.layers.Select
+import com.intel.analytics.bigdl.tensor.Tensor
+
+val model = Sequential[Float]()
+model.add(Select[Float](1, 2))
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: where's inputShape?

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 22]
```diff
@@ -283,3 +283,312 @@ Output is
...
+
+**Python example:**
+```python
+from zoo.pipeline.api.keras.layers import Select
+from zoo.pipeline.api.keras.models import Sequential
+import numpy as np
+
+model = Sequential()
+model.add(Select(1, 2, input_shape=(3, 3)))
+input = np.random.random([3, 3])
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: if you specify input_shape=(3, 3) then the input should be 3D...

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 23]
```diff
@@ -283,3 +283,312 @@ Output is
...
+```python
+Dense(output_dim, init="glorot_uniform", activation=None, W_regularizer=None, b_regularizer=None, bias=True, input_dim=None, input_shape=None, name=None)
+```
+
+**Parameters:**
+
+* `outputDim`: The size of the output dimension.
+* `init`: Initialization method for the weights of the layer. Default is Xavier.You can also pass in corresponding string representations such as 'glorot_uniform'or 'normal', etc. for simple init methods in the factory method.
+* `activation`: Activation function to use. Default is null.You can also pass in corresponding string representations such as 'relu'or 'sigmoid', etc. for simple activations in the factory method.
+* `wRegularizer`: An instance of [Regularizer](../../APIGuide/Regularizers/), applied to the input weights matrices. Default is null.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: link 404

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 24]
```diff
@@ -283,3 +283,312 @@ Output is
...
+## **Negative**
+Computes the negative value of each element of the input.
+
+**Scala:**
+```scala
+Negative(inputShape = null)
+```
+**Python:**
+```python
+Negative(input_shape = None)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: missing name?

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 25]
```diff
@@ -283,3 +283,312 @@ Output is
...
+
+The expand means repeat on unmatched singleton dimension (if some unmatched dimension isn't a singleton dimension, an error will be raised).
+
+**Scala:**
+```scala
+CAdd(size, bRegularizer = null, inputShape = null)
+```
+**Python:**
+```python
+CAdd(size, bRegularizer = None, bigdl_type = "float", input_shape = None, name = None)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: don't include `bigdl_type` in ALL layer arguments...

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 26]
```diff
@@ -283,3 +283,312 @@ Output is
...
+
+**Scala example:**
+```scala
+import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
+import com.intel.analytics.zoo.pipeline.api.keras.layers.CAdd
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.tensor.Tensor
+
+val model = Sequential[Float]()
+model.add(CAdd[Float](Array(2, 3), bRegularizer=null, inputShape = Shape(2, 3)))
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: remove bRegularizer=null... as it it default to be null

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 27]
```diff
@@ -283,3 +283,312 @@ Output is
...
+```
+
+**Python example:**
+```python
+from zoo.pipeline.api.keras.layers import CAdd
+from zoo.pipeline.api.keras.models import Sequential
+import numpy as np
+
+model = Sequential()
+model.add(CAdd([2, 1], bigdl_type="float", input_shape = (2, 3)))
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: remove bigdl_type

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 28]
```diff
@@ -283,3 +283,312 @@ Output is
...
+**Scala example:**
+```scala
+import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
+import com.intel.analytics.zoo.pipeline.api.keras.layers.CAdd
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.tensor.Tensor
+
+val model = Sequential[Float]()
+model.add(CAdd[Float](Array(2, 3), bRegularizer=null, inputShape = Shape(2, 3)))
+val input = Tensor[Float](2, 3).rand()
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: should be 3D input

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 29]
```diff
@@ -283,3 +283,312 @@ Output is
...
+
+**Python example:**
+```python
+from zoo.pipeline.api.keras.layers import CAdd
+from zoo.pipeline.api.keras.models import Sequential
+import numpy as np
+
+model = Sequential()
+model.add(CAdd([2, 1], bigdl_type="float", input_shape = (2, 3)))
+input = np.random.rand(2, 3)
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: input shape and actual input doesn't match. input shape is actual tensor shape without batch

File: docs/docs/KerasStyleAPIGuide/Layers/dropout.md [hunk 30]
```diff
@@ -0,0 +1,91 @@
...
+
+**Parameters:**
+
+* `p`: Fraction of the input units to drop. Between 0 and 1.
+* `inputShape`: Only need to specify this argument when you use this layer as the first layer of a model. For Scala API, it should be a [`Shape`](../keras-api-scala/#shape) object. For Python API, it should be a shape tuple. Batch dimension should be excluded.
+
+**Scala example:**
+```scala
+import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
+import com.intel.analytics.zoo.pipeline.api.keras.layers. Dropout
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/dropout.md]: remove extra space

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 12 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: OK

File: pyzoo/zoo/pipeline/api/keras/layers/normalization.py (hunk 14 above)
Comment (LudviqIntel) [reply pyzoo/zoo/pipeline/api/keras/layers/normalization.py]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md (hunk 15 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md (hunk 16 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md (hunk 17 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md (hunk 18 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 20 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 24 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, fix

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 25 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK，fix

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 26 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, fix

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 27 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, fixed

File: docs/docs/KerasStyleAPIGuide/Layers/dropout.md (hunk 30 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/dropout.md]: OK, fixed

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 29 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, fixed

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 28 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, fixed

File: pyzoo/zoo/pipeline/api/keras/layers/normalization.py (hunk 14 above)
Comment (LudviqIntel) [reply pyzoo/zoo/pipeline/api/keras/layers/normalization.py]: fixed

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md (hunk 19 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: OK, fixed

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 21 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, fixed

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 22 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, fixed

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 23 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: OK, changed

File: docs/docs/KerasStyleAPIGuide/Layers/dropout.md [hunk 31]
```diff
@@ -0,0 +1,91 @@
...
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 2x3x4]
+```
+
+**Python example:**
+```python
+import numpy as np
+from zoo.pipeline.api.keras.layers import Dropout
+from zoo.pipeline.api.keras.models import Sequential
+
+domodel = Sequential()
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/dropout.md]: what is domodel?

**Comment by hkvision** [comment]
LGTM besides the comments above.

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 32]
```diff
@@ -1263,3 +1263,319 @@ Output is
...
+
+---
+## **Select**
+Select an index of the input in the given dim and return the subset part.
+
+The batch dimension needs to be unchanged.
+
+For example, if input is:
+
+1 2 3
```
[the remark this answers is not in the document]
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: ok,fixed

File: docs/docs/KerasStyleAPIGuide/Layers/dropout.md (hunk 31 above)
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/dropout.md]: fixed,renamed to model

File: docs/docs/KerasStyleAPIGuide/Layers/activation.md [hunk 33]
```diff
@@ -176,3 +176,81 @@ Output is
...
+* `lower`: Lower boundary of the uniform random distribution. Default is 1.0/8.
+* `upper`: Upper boundary of the uniform random distribution. Default is 1.0/3.
+* `inputShape`: Only need to specify this argument when you use this layer as the first layer of a model. For Scala API, it should be a [`Shape`](../keras-api-scala/#shape) object. For Python API, it should be a shape tuple. Batch dimension should be excluded.
+
+**Scala example:**
+```scala
+import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
+import com.intel.analytics.zoo.pipeline.api.keras.layers.RReLU
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.tensor.Tensor
```
[the remark this answers is not in the document]
Comment (LudviqIntel) [reply docs/docs/KerasStyleAPIGuide/Layers/activation.md]: ok, fixed and checked else

**Pull request merged**